packages/web-components/src/components/rux-timeline/rux-track/rux-track.tsx

```diff
@@ -171,24 +185,24 @@ export class RuxTrack {
         this.initializeRows()
     }
 
-    renderDebug() {
-        return (
-            <div style={{ display: 'contents' }}>
-                {[...Array(this.columns)].map((_: any, i: any) => (
-                    <div
-                        style={{
-                            gridRow: '1',
-                            gridColumn: `${i + 2} / ${++i + 2}`,
-                        }}
-                        class={{
-                            cell: true,
-                            marker: i % 60 === 0,
-                        }}
-                    ></div>
-                ))}
-            </div>
-        )
-    }
+    // renderDebug() {
+    //     return (
+    //         <div style={{ display: 'contents' }}>
+    //             {[...Array(this.columns)].map((_: any, i: any) => (
+    //                 <div
+    //                     style={{
+    //                         gridRow: '1',
+    //                         gridColumn: `${i + 2} / ${++i + 2}`,
+    //                     }}
+    //                     class={{
+    //                         cell: true,
+    //                         marker: i % 60 === 0,
+    //                     }}
+    //                 ></div>
+    //             ))}
+    //         </div>
+    //     )
+    // }
```

Comment:
Would you leave a `/** @TODO` note if you want to keep this commented-out code? Non-blocking.